_config.yml

```diff
@@ -5,7 +5,8 @@ description: > # this means to ignore newlines until "baseurl:"
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://akshattripathi.github.io/fossasia14/" # the base hostname & protocol for your site
+url: "https://shashwat1002.github.io/2014.fossasia.org/" # the base hostname & protocol for your site
```

Comment:
Add the actual path of the website.

Which is `http://fossasia.github.io/2014.fossasia.org/`